src/Kanbas/Courses/Quizzes/Editor/QuestionsTab.tsx

```diff
@@ -0,0 +1,135 @@
+import { useEffect, useState } from "react";
+import "./QuestionsTab.css";
+import { FaBan, FaCheck, FaPlus, FaTrash } from "react-icons/fa";
+import { FaMagnifyingGlass } from "react-icons/fa6";
+import { Question, QuestionType } from "../types";
+import QuestionEditor from "./QuestionEditor";
+
+function QuestionsTab() {
+  const [points, setPoints] = useState(0);
+  const [published, setPublished] = useState(false);
+  const [notifyUsers, setNotifyUsers] = useState(false);
+  const [questions, setQuestions] = useState<Question[]>([]);
+  const [initialQuestions, setInitialQuestions] = useState<Question[]>([]);
+
+  // TODO: PULL QUESTIONS FROM DB ?
+  useEffect(() => {
+    setQuestions([]);
+    setInitialQuestions([]);
+  }, []);

```

Comment:
is this not already added in the backend that Jess has already pushed?

Reply:
oh yeah I was waiting for the cluster to be up to actually implement + test these, I can make a new PR for that